docs/_documentations/project-actions.md

```diff
@@ -6,35 +6,28 @@ keywords: run, open, import, show, restart, edit, build, logs, tools, project ac
 duration: 1 minute
 permalink: project-actions
 type: document
-order: 30
-parent: 
 ---
 
 # Project actions
 
-The following actions are available by right clicking on the project in the **Codewind Explorer** view. Most actions are only available if the project is enabled.
-
-Some actions open the default browser. If you find that the default browser cannot handle the content, change the default browser by navigating to **Window** > **Web Browser** and selecting a different browser from the list.
-
-- **Open Application**: Opens the application in the default browser. This action is only available when the application is running or debugging.
-- **Project Overview**: Opens the overview page for this project. You can use this to see information about the project or edit project settings.
-- **Metrics Dashboard**: Opens the application monitor in the default browser. Use this to monitor the activity and health of your application. This action is only available when the application is running or debugging.
```

Comment:
At the project level, we see “Metrics Dashboard” option in Java, Node.js, and Swift applications: 
- **Metrics Dashboard**:  Opens the Metrics Dashboard in the default browser. Use this to monitor the activity and health of your application. This action is only available when the application is running or debugging, and it is only supported in Java, Node.js, and Swift templates.